docs/api/config.md

```diff
@@ -258,7 +258,9 @@ By default, this is not enabled. You can enable this by setting it to `true`.
 
     This is an experimental feature and is subject to breaking change. Please follow [https://github.com/google/mesop/issues/1028](https://github.com/google/mesop/issues/1028) for updates.
 
-This uses WebSockets instead of HTTP Server-Sent Events (SSE) as the transport protocol for UI updates. If you set this environment variable to `true`, then [`MESOP_CONCURRENT_UPDATES_ENABLED`](#MESOP_CONCURRENT_UPDATES_ENABLED) will automatically be enabled as well.
+This uses WebSockets instead of HTTP Server-Sent Events (SSE) as the transport protocol for UI updates. Enabling this also enables user events to be handled concurrently, which is particularly useful supporting interactivity for long-running event handlers.
```

Comment:
Maybe adjust the wording a bit for Enabling this also enables...

Sounds a bit odd now that we've deprecating MESOP_CONCURRENT_UPDATES_ENABLED. But can update when it's fully removed.

Reply:
slightly reworded